src/content/platform/limits.md

```diff
@@ -19,6 +19,7 @@ order: 2
 | [Environment variable<br/>size](#environment-variables)                         | 5 KB                                                 | 5 KB      |
 | [Script size](#script-size)                                                     | 1 MB                                                 | 1 MB      |
 | [Number of scripts](#number-of-scripts)                                         | 30                                                   | 30        |
+| [Number of Cron Triggers<br/>per script](#number-of-schedules)                  | 3                                                    | 3         |
```

Comment:
is this per acct?

Reply:
also i think the br tag should be removed

Reply:
The `<br/>` tag is OK for now. It helps with the position sticky first column the TableWrap uses on small displays.

![image](https://user-images.githubusercontent.com/154613/94098556-dd038900-fdf6-11ea-894a-13a272f9936b.png)

(scrolled to the right)

![image](https://user-images.githubusercontent.com/154613/94098578-e856b480-fdf6-11ea-91c0-2313e26529f6.png)

It’s not ideal but it’s a decent “general” solution that works for now.

Reply:
got it. the link is breaking on markdown preview here but if we're confident it renders with that br i'm good w/ this

Reply:
Yes, it renders nicely and matches other things in the list.